tooling/miniutil/src/build/mod.rs

```diff
@@ -223,6 +223,10 @@ pub fn size(bytes: impl Into<Int>) -> Size {
     Size::from_bytes(bytes).unwrap()
 }
 
+pub fn sized_size(bytes: impl Into<Int>) -> SizeStrategy {
```

Comment:
That's an odd function name -- and also an unused function, it seems?

Reply:
Ah yes indeed. Used to be used in some tests, before they got rewritten to use TypeConv by another PR.